cmd/minishift/cmd/start.go

```diff
@@ -87,6 +82,13 @@ var minishiftToClusterUp = map[string]string{
 
 // init configures the command line options of this command
 func init() {
+	startCmd = &cobra.Command{
```

Comment:
@hferentschik Right, wondering why is this so. `runStart` anyways runs after `init()`.